README.md

```diff
@@ -75,7 +75,37 @@ Now you're ready to [run the application](#run-the-application)!
 <a id="use-a-docker-container"></a>
 ## Use a Docker container
 
-COMING SOON!
+We've created a [Docker container](ghcr.io/reboot-dev/resemble-standalone) that _has all of the dependencies you need to build and run code in this repository already installed_.
+> [!IMPORTANT]
+> The Docker container currently **only works on x86 CPU architectures**. Check back soon for more supported architectures.
+
+Clone this repository:
+
+```shell
+git clone https://github.com/reboot-dev/resemble-examples.git
+cd resemble-examples/
+```
+
+Run the container:
+
+```shell
+docker run \
+  --mount type=bind,source=$PWD,target=/workspaces/resemble \
+  -p 127.0.0.1:3000:3000/tcp \
+  -p 127.0.0.1:9991:9991/tcp \
+  --privileged \
+  --interactive \
+  --tty \
+  ghcr.io/reboot-dev/resemble-standalone:latest
+```
+
+Explanation of flags:
+* We mount in our git cloned directory so that we won't lose any of our changes.
+* We bind port `3000` so that we can access the web front end and port `9991` so the web front end can access the backend.
```

Comment:
```suggestion
* We bind port `3000` so that we can access the web front end (e.g., from a browser), and port `9991` so the web front end can access the backend.
```